PERFORMANCE

```diff
@@ -37,6 +37,55 @@ To characterize Chapel performance, generally speaking...
   assignments are used to move chunks of data between locales.
 
 
+Experimental flags for improving performance
+--------------------------------------------
+
+The following compiler flags are intended to provide a preview of
```

Comment:
Is 'compiler flags' the right term? I'm not opposed to the wording, but I tend to think of compiler flags as things that would be printed out with `chpl --help`.